src/ECER.Tests/Integration/RegistryApi/CommunicationTests.cs

```diff
@@ -39,6 +39,32 @@ public async Task GetMessageStatus_ReturnsStatus()
 
     var communicationsStatus = await communicationsStatusResponse.ReadAsJsonAsync<CommunicationsStatusResults>();
     communicationsStatus.ShouldNotBeNull();
-
+  }
+  
+  
+  [Fact]
+  public async Task SeenCommunication_ReturnId()
+  {
+    var communicationSeenResponse =  await Host.Scenario(_ =>
```

Comment:
This Test will work just one time

Reply:
@farzadnadiri @SoLetsDev this test is **not** passing consistently